### PR TITLE
Reintroduce kickers onto paid cards with same style as rich links in paid content

### PR DESCRIPTION
**Pull request by JonNorman**
## What does this change?
Commercial containers and cards were refactored a few months ago. During this refactor the ability to add kickers to commercial cards - i.e. paid-for or sponsored content- in containers was removed. It turns out that this is useful, so this PR (re)introduces this functionality.

> Erm, excuse me but what's a kicker?

A kicker is a short tagline that appears before (and inline with) the headline of a card.

Input from design is that these kickers should look the same as those on [rich-links within paid-for articles](https://www.theguardian.com/progress-personified/2016/nov/08/mount-everest-wounded-veteran-amputee-mountaineer-marine).

## What is the value of this and can you measure success?
This allows Central Production to better style the cards and improve brand awareness. It is also a prerequisite for launching paid-for containers on fronts.

## Screenshots
**before**
image

**after**
image

## Request for comment
@guardian/commercial-dev

### CODE REVIEW

**Comment by JonNorman** [comment]
[GLabs add kicker to fronts container](https://trello.com/c/1dgKcMhD/143-glabs-add-kicker-to-fronts-container)

**Comment by rich-nguyen** [comment]
nice 👍

File: static/src/stylesheets/module/commercial/_advert.scss [hunk 1]
```diff
@@ -115,6 +115,18 @@
     }
 }
 
+.advert__kicker {
+    &:after {
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_advert.scss]: `&::after`

File: static/src/stylesheets/module/commercial/_advert.scss [hunk 2]
```diff
@@ -115,6 +115,18 @@
...
+        content: '/';
+        display: inline-block;
+        margin-left: .2em;
+        color: mix(#ffffff, $news-support-6, 20%);
```
Comment (regiskuckaertz) [inline static/src/stylesheets/module/commercial/_advert.scss]: 👏  😮

**Comment by regiskuckaertz** [comment]
Sharp!

File: static/src/stylesheets/module/commercial/_advert.scss (hunk 2 above)
Comment (JonNorman) [inline static/src/stylesheets/module/commercial/_advert.scss]: Yeah, took  a bit of time to track down where the actual RGB colour on the paid-for rich links was coming from :weary:

File: static/src/stylesheets/module/commercial/_advert.scss (hunk 1 above)
Comment (JonNorman) [inline static/src/stylesheets/module/commercial/_advert.scss]: Ah good to know - just [read up on this](http://stackoverflow.com/a/11701951) - fairly subtle!

**Pull request merged**